src/renderer/index.js

```diff
@@ -62,6 +62,18 @@ ipcRenderer.on('window-is-focused', () => {
   dock.setBadge('');
 });
 
+'use strict';
+
+document.addEventListener('dragover', function (event) {
+  event.preventDefault();
+  return false;
+}, false);
```

Comment:
You have some extra stuff that isn't needed.
```
document.addEventListener('dragover', event => {
  event.preventDefault();
})
```

Reply:
i remove that and test on my local setup